doc/source/whatsnew/v0.25.0.rst

```diff
@@ -254,6 +254,7 @@ Performance Improvements
 - Improved performance of :meth:`Series.map` for dictionary mappers on categorical series by mapping the categories instead of mapping all values (:issue:`23785`)
 
 .. _whatsnew_0250.bug_fixes:
+- Fixed MultiIndex DataFrame to_csv segfault (:issue:`26303`)
```

Comment:
move to Indexing, use ``:class:`MultiIdex` and ``:meth:`DataFrame.to_csv` ``; specify that this is a single-level multi-index

Reply:
Done in 7265dc2a0252b552d954fa72d8ddbf06618efef4

Reply:
Done in d47089ee3ef92152f9577602b84073455bb49763